compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1936,6 +1936,18 @@ OMR::Power::CodeGenerator::addMetaDataForLoadAddressConstantFixed(
          TR::DebugCounter::generateRelocation(comp, firstInstruction, node, counter, seqKind);
          return;
          }
+
+      case TR_BlockFrequency:
+	 {
+         TR_RelocationRecordInformation *recordInfo = ( TR_RelocationRecordInformation *)comp->trMemory()->allocateMemory(sizeof(TR_RelocationRecordInformation), heapAlloc);
+         recordInfo->data1 = (uintptr_t)node->getSymbolReference();
+         recordInfo->data2 = (uintptr_t)seqKind;
+         relo = new (self()->trHeapMemory()) TR::BeforeBinaryEncodingExternalRelocation(
+            firstInstruction,
+            (uint8_t *)recordInfo,
+            TR_BlockFrequency, self());
+         break;
+         }
```

Comment:
How come you don't also need a relo record for `TR_RecompQueuedFlag` here?

Reply:
`TR_RecompQueuedFlag` gets taken care by the default case towards the end of the method:
```
   if (!relo)
      {
      relo = new (self()->trHeapMemory()) TR::BeforeBinaryEncodingExternalRelocation(
         firstInstruction,
         (uint8_t *)value,
         (uint8_t *)seqKind,
         (TR_ExternalRelocationTargetKind)typeAddress,
         self());
      }
```